### ISSUE TITLE
Error calling cluster events

### ISSUE THREAD

**Issue by jcolares** [gbrueckl/Databricks.API.PowerShell#15]
Get-DatabricksClusterEvent -Cluster_Id "xxxx-xxxxxx-xxxxxxxx" returns the error below:
```
Invoke-RestMethod : {"error_code":"ENDPOINT_NOT_FOUND","message":"No API found for 'GET /clusters/events'"}
At C:\Program Files\WindowsPowerShell\Modules\DatabricksPS\0.9.3.2\Public\General.ps1:53 char:12
+ ...   $result = Invoke-RestMethod -Uri $apiUrl -Method $Method -Headers $ ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand

```
The solution would be to change the request method "GET" to "POST" in ClustersAPI.ps1, line 591:`
$requestMethod = "POST"`

**Comment by gbrueckl** [comment]
Hi @jcolares 
thanks for the info, will fix it and publish a new version to the gallery in a few hours!

btw, for the future, you can simply create a Pull Request and do the change on your own so I just have to approve it!

regards,
-gerhard

**Comment by gbrueckl** [comment]
just published Version 9.3.3 which fixes this issue
thanks again for reporting!
